src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -40,7 +82,22 @@ private void handleCommandEntered() {
         if (commandText.equals("")) {
             return;
         }
```

Comment:
This may have some slightly unexpected behavior.
Consider the following case:
1. I copy command A and B onto my clipboard.
2. I paste the 2 cmds into the cmd line, and the queue is now `[A, B]`.
3. I clear everything from the cmd line (backspace all).
4. I type in `A` again and press enter.
5. Then I see `B`, which may not be the expected outcome.

Reply:
> When multiple lines of commands are pasted into the box, pressing enter will execute the first command and autofill the next one. This continues until a different command is entered or all commands have been entered.

I would say it's consistent with the documentation. I think how much we polish this depends on how much we see this as a legit feature instead of something to assist with testing. That's something we should discuss